src/tests/bulkConcepts/containers/BulkConceptsPage.test.js

```diff
@@ -297,7 +297,58 @@ describe('Test suite for BulkConceptsPage component', () => {
     wrapper.find('#add-button').simulate('click');
     jest.runAllTimers();
   });
-
+  it('should check whether the CIEL concept has answer mappings', () => {
```

Comment:
You have multiple instances of should handle, down below.
Secondly, can you point me to the code, being tested (not the test), that checks whether CIEL concepts have mappings?

Reply:
> You have multiple instances of should handle, down below.

I believe you looked at the code as I was making changes. IT should have been rectified
 
> Secondly, can you point me to the code, being tested (not the test), that checks whether CIEL concepts have mappings?

`src/components/bulkConcepts/component/ActionButtons.jsx`. Lines 47 to 67

Reply:
> You have multiple instances of should handle, down below.

I believe you looked at the code as I was making changes. IT should have been rectified
 
> Secondly, can you point me to the code, being tested (not the test), that checks whether CIEL concepts have mappings?

`src/components/bulkConcepts/component/ActionButtons.jsx`.

Reply:
Is this test about ensuring that when a concept does not have answer mappings then it should not save it?

Reply:
No, this test is about checking whether a concept has an answer mapping or not. The logic that handles saving of the information is in a completely separate file

Reply:
What happens when it does not have an answer mapping?

Reply:
Additionally, if a concept has a mapping, it should be saved. If it does not have a mapping, it should still be saved

Reply:
Then in that case, what is the use of this test?

Reply:
This test simply confirms that the code we have written to check that whether an incoming concept has a mapping or not is working as intended.

Reply:
Can you discuss this with your team mates?

Reply:
Sorry, I edited the comment and had not seen the above message. Could you confirm whether it my explanation is satisfactory?

Reply:
You test description does not reflect that

Reply:
I liaised with the team and have edited the description to be more specific to checking how the recursive methods handle the concept data they receive. Given that those were the missing lines and this is exactly what the test is testing, is the description sufficient?

Reply:
Before i take a second look at the description, does the test fail, if you remove the lines (Lines 47 to 67 in ActionButtons) that you claim to be testing?

Reply:
No, let me look into this and get back to you

Reply:
Then it is not testing anything.